app/login_window.py

```diff
@@ -64,7 +64,7 @@ def login(self):
                                         self.username.text()+
                                         '&password='+self.password.text())
                 login_status = json.loads(login_status.text)
-                if login_status.get('status', False):
+                if login_status.get('status', True):
```

Comment:
@haridinesh00 why have you set the default value to True? So, even if the login fails, the app would still open.

Reply:
With the value false, the login doesn't work anymore, even with the right credentials. With the value set to True, if a wrong credential is entered it says login failed and doesn't proceed to the next screen. Please run the code and see for yourself.
With these minor changes, I was able to get the code up and running smoothly. 
Thanks for replying, you have really helped us with our project.

Reply:
Have you checked it by running? because it is working as I expected. The login_status is of type Dict and if login fails the API returns nothing. So, I'm adding a default value of False to logic_status which makes sure the app doesn't open when the login fails. What you are doing is setting its value to True which would mean if login_status doesn't have a status key then it would return True which satisfies if condition.